pylint_odoo/checkers/modules_odoo.py

```diff
@@ -91,7 +91,7 @@
 # Files supported from manifest to convert
 # Extracted from openerp/tools/convert.py:def convert_file
 DFLT_EXTFILES_CONVERT = ['csv', 'sql', 'xml', 'yml']
-DFLT_EXTFILES_TO_LINT = DFLT_EXTFILES_CONVERT + ['po', 'js', 'mako']
+DFLT_EXTFILES_TO_LINT = DFLT_EXTFILES_CONVERT + ['po', 'js', 'mako', 'rst']
```

Comment:
Added `md` and `markdown` extension to extra files to lint.
I don't understood the `png` files because they are binary (not text) then I can't lint a binary file.